params/protocol_params.go

```diff
@@ -207,12 +206,15 @@ var (
 // This is TimeFactor*TimeFactor*common.NumZonesInRegion*common.NumRegionsInPrime
 func PrimeEntropyTarget(expansionNum uint8) *big.Int {
 	numRegions, numZones := common.GetHierarchySizeForExpansionNumber(expansionNum)
-	timeFactorMultiplied := new(big.Int).Mul(TimeFactor, TimeFactor)
+	regionTimeFactor := int64(max(numRegions, 2))
+	zoneTimeFactor := int64(max(numZones, 2))
+	timeFactorMultiplied := new(big.Int).Mul(big.NewInt(regionTimeFactor), big.NewInt(zoneTimeFactor))
 	return new(big.Int).Mul(timeFactorMultiplied, new(big.Int).SetUint64(numZones*numRegions))
 }
 
 // This is TimeFactor*common.NumZonesInRegion
 func RegionEntropyTarget(expansionNum uint8) *big.Int {
 	_, numZones := common.GetHierarchySizeForExpansionNumber(expansionNum)
-	return new(big.Int).Mul(TimeFactor, new(big.Int).SetUint64(numZones))
+	timeFactor := int64(max(numZones, 2))
```

Comment:
Pedantic but just set this to be a big int instead of creating an int64 first and then another big.int